bibliofil_lists.module

```diff
@@ -0,0 +1,293 @@
+<?php
+
+/**
+ * @file
+ * Bibliofil lists module.
+ */
+
+define('DYNPATH', '/cgi-bin/m2dyn.htmc');
+define('WEBLISTERPATH', '/cgi-bin/rest_service/web_lister/1.0/data/');
+
+/**
+ * Implements hook_ctools_plugin_directory().
+ */
+function bibliofil_lists_ctools_plugin_directory($owner, $plugin_type) {
+  if ($owner == 'ctools' || $owner == 'panels') {
+    return 'plugins/' . $plugin_type;
+  }
+}
+
+/**
+ * Implements hook_menu().
+ */
+function bibliofil_lists_menu() {
+  $items = [];
+
+  $items['admin/config/bibliofil'] = [
+    'title' => 'Bibliofil',
+    'description' => 'Bibliofil specific settings',
+    'page callback' => 'system_admin_menu_block_page',
+    'access arguments' => ['access administration pages'],
```

Comment:
Use the `administer site configuration` permissions. As this one stands for actually granting access to changing configuration.
`access administration pages` is used to simply grant page view.